calico-enterprise/release-notes/index.mdx

```diff
@@ -14,11 +14,11 @@ Added support for VXLAN overlay networking to federated endpoint indentity and s
 
 For more information, see [Configure federated endpoint identity and services](../multicluster/federation/kubeconfig).
 
-### Feature two
+### Helm install and upgrade for multi-cluster management
 
-Here's what you need to know about this feature.
+Adds Helm install and upgrade for {{prodname}} multi-cluster management. As a de facto standard for deploying software in Kubernetes, Helm provides deployment automation in tools like Gitlab.
```

Comment:
The second sentence reads a bit sales-y. Perhaps something like:

We've added the option to use Helm to install and upgrade {{prodname}} in multi-cluster settings.
With Helm, your DevOps teams can automate these complex deployments more easily.

Reply:
Fixed.